pkg/component/configuration_grpc.go

```diff
@@ -44,11 +44,11 @@ func (c *ConfigurationServer) RegisterServices(s *grpc.Server) {
 
 // RegisterHandlers registers the Configuration service handler.
 func (c *ConfigurationServer) RegisterHandlers(s *runtime.ServeMux, conn *grpc.ClientConn) {
-	ttnpb.RegisterConfigurationHandler(c.component.Context(), s, conn)
+	_ = ttnpb.RegisterConfigurationHandler(c.component.Context(), s, conn)
 }
 
 // ListFrequencyPlans implements the Configuration service's ListFrequencyPlans RPC.
-func (c *ConfigurationServer) ListFrequencyPlans(ctx context.Context, req *ttnpb.ListFrequencyPlansRequest) (*ttnpb.ListFrequencyPlansResponse, error) {
+func (c *ConfigurationServer) ListFrequencyPlans(ctx context.Context, req *ttnpb.ListFrequencyPlansRequest) (*ttnpb.ListFrequencyPlansResponse, error) { //nolint:lll
```

Comment:
Do not use `nolint:lll` here and below.

When the signatures get too big, we prefer to use the following rule set:
1. Give the parameters their own individual line, like so:
```
func (c *ConfigurationServer) ListFrequencyPlans(
	ctx context.Context, req *ttnpb.ListFrequencyPlansRequest,
) (*ttnpb.ListFrequencyPlansResponse, error) {
```
2. If this is still too long, give each parameter a new line:
```
func (c *ConfigurationServer) ListFrequencyPlans(
	ctx context.Context,
	req *ttnpb.ListFrequencyPlansRequest,
) (*ttnpb.ListFrequencyPlansResponse, error) {
```

Same rules for return values.

Reply:
Tried that but found it not helping the readability. But if that's the standard I'll gladly follow it